crates/dht/src/dht.rs

```diff
@@ -57,16 +61,18 @@ pub enum DhtSnapshotNextAction {
 }
 
 impl Dht {
-    /// Create a new DHT from an op store.
+    /// Create a new DHT instance from an op store.
     ///
-    /// Creates the inner [PartitionedHashes] using the store.
+    /// Creates the inner [HashPartition] using the store. The sizing for the sectors and time
+    /// slices are currently hard-coded. This will create 512 sectors and each full time slice will
+    /// be approximately 5.3 days.
     pub async fn try_from_store(
         current_time: Timestamp,
         store: DynOpStore,
     ) -> K2Result<Dht> {
         Ok(Dht {
-            partition: PartitionedHashes::try_from_store(
-                14,
+            partition: HashPartition::try_from_store(
+                9,
```

Comment:
This was a mistake on my part. 14 creates seriously large full time slices. I'd originally documented 9 intending to use that. It means you get full time slices that are about 5 days.